ts/utils/dates.ts

```diff
@@ -124,3 +124,21 @@ export class DateFromISOStringType extends t.Type<Date, string, unknown> {
 }
 
 export const DateFromISOString: DateFromISOStringType = new DateFromISOStringType();
+
+export const getTranslatedShortNumericMonthYear = (
```

Comment:
Could you leave a comment?

Reply:
Sure! Added in [b4bd832](https://github.com/pagopa/io-app/pull/2926/commits/b4bd83230e05070a16d30f17b9ea1e7124193d8b)